tasks/collect-slack-notification-params/README.md

```diff
@@ -0,0 +1,15 @@
+# collect-slack-notification-params
+
+Tekton task that collects slack configuration options from Release CRs and the data file. A formatted message is emitted
+as a task result for downstream tasks to use.
+
+## Parameters
+
+| Name                 | Description                                                                                                  | Optional | Default value                                                      |
+|----------------------|--------------------------------------------------------------------------------------------------------------|----------|--------------------------------------------------------------------|
+| release              | The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution       | No       | -                                                                  |
+| releaseplan          | The namespaced name (namespace/name) of the releasePlan                                                      | No       | -                                                                  |
```

Comment:
It seems that RP and RPA are not actually there? Neither in the task yaml, nor in the call of the task in the pipeline.

Reply:
fixed